guides/source/active_storage_overview.md

```diff
@@ -537,6 +537,13 @@ Serving Files
 
 Active Storage supports two ways to serve files: redirecting and proxying.
 
+WARNING: The default Active Storage controllers are publicly accessible.
+The generated URLs are hard to guess, but anyone that knows the blob
+URL will be able to download it, even if a `before_action` in your
+`ApplicationController` would otherwise require a login. If your files require
+a higher level of protection consider implementing [Authenticated
+Controllers](active_storage_overview.html#authenticated-controllers).
```

Comment:
This section duplicates a lot of the wording from the `Authenticated Controllers` section, I think we can shorten this part and expand upon it below.

Reply:
:+1:
I was afraid of removing too much information, but managed too shorten it.